march_joint_inertia_controller/include/march_joint_inertia_controller/inertia_estimator.h

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 Project March.
+
+#ifndef MARCH_WS_INERTIA_ESTIMATOR_H
+#define MARCH_WS_INERTIA_ESTIMATOR_H
+
+#include <hardware_interface/joint_command_interface.h>
+#include <ros/ros.h>
+#include <trajectory_interface/quintic_spline_segment.h>
+#include <urdf/model.h>
+
+/**
+ * \brief determines the absolute value of vector a and stores it into b so that a remains unchanged
+ *
+ * @param[in] a input vector with values that need to be converted to absolutes
+ * @param[out] b output vector with copies of the absolute values of a.
+ */
+void absolute(std::vector<double> a, std::vector<double>& b);
```

Comment:
Why not just return the vector? Then you don't have to initialize a vector first before calling this and this is what return values are for. Furthermore, it is best to pass a vector as a const reference if you don't want to modify it, since it does not require a copy.
```suggestion
std:vector<double> absolute(const std::vector<double>& a);
```